src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -296,7 +296,7 @@ describe('workspace service', () => {
         .expect(200);
       expect(findResult.body.total).toEqual(0);
       // Global and Management workspace will be created by default after workspace list API called.
```

Comment:
Could you please remove the `Management` in the comment?

Reply:
done